### DISCUSSION TITLE
posts/Slew-Rate-Control/

### DISCUSSION THREAD

**Discussion by giscus** [847X-Robotics/847X-Robotics.github.io#8] [General]
# posts/Slew-Rate-Control/

The Problem The search for “perfect” acceleration is trying to find a balance of getting to the desired speed quickly while also avoiding tipping, wheel slippage, or excessive strain on the motor. As robots are heavy and VEX motors are only so strong, trying to go from rest to full speed immediately will result in jerky movements at least, or tipping and even rolling forwards/backwards on robots with poor COG. Additionally, if motor encoders are being used for any kind of tracking or positioning, accelerating too fast typically leads to wheel slippage, producing inaccurate measurements, not to mention over time wears on the motors. In particular, when relying on motor encoders as tracking wheels, it’s essential to reduce wheel slippage as much as possible, risking large inconsistencies if not done. For all these reasons, it might be clear why we often try to find ways to limit the acceleration of our movemenets.

https://847x-robotics.github.io/posts/Slew-Rate-Control/

**Comment by meisZWFLZ** [comment]
Yooo comments

**Reply by edwart2cAKE** [reply]
W comments!
